src/applications/apps.js

```diff
@@ -415,6 +415,10 @@ export function pathToActiveWhen(path, exactMatch) {
   const regex = toDynamicPathValidatorRegex(path, exactMatch);
 
   return (location) => {
+    // compatible with IE10
+    if (!location.origin) {
+      location.origin = `${location.protocol}//${location.hostname}${location.port ? ':' + location.port: ''}`;
```

Comment:
I'd prefer not overwriting location.origin, but just creating a variable that we use later.

```suggestion
      const origin = `${location.protocol}//${location.hostname}${location.port ? ':' + location.port: ''}`;
```

Reply:
Done!